### ISSUE TITLE
[Question] Has anyone tried multinode training (LLM finetuning) with FSDP?

### ISSUE THREAD

**Issue by kahnchana** [haotian-liu/LLaVA#295]
### Question

Can the current setup be directly used on multinode setup? Using slurm and torchrun?

**Comment by kahnchana** [comment]
It works out of the box with a slurm scheduler. 

Simply appending following to torchrun in the train scripts: 
```
srun --jobid $SLURM_JOBID \
    torchrun \
    --nnodes=$SLURM_NNODES \
    --nproc_per_node=8 \
    --node_rank $SLURM_PROCID \
    --rdzv_id=1 \
    --rdzv_backend=c10d \
    --rdzv_endpoint="${MASTER_ADDR}:${MASTER_PORT}" \
    llava/train/train_mem.py . . . 
```

**Comment by yuvalkirstain** [comment]
@kahnchana can you please write here the full command? I am now looking into this.
